docs/_includes/generated/environment_configuration.html

```diff
@@ -0,0 +1,41 @@
+<table class="table table-bordered">
+    <thead>
+        <tr>
+            <th class="text-left" style="width: 20%">Key</th>
+            <th class="text-left" style="width: 15%">Default</th>
+            <th class="text-left" style="width: 65%">Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><h5>env.java.opts</h5></td>
+            <td>(none)</td>
+            <td></td>
```

Comment:
We could add a short descriptions about what these `opts` things actually do.

Reply:
There are plenty of missing descriptions, but I wanted to do that as a follow-up to get the basic setup in place first.

Reply:
KK, go ahead, then. 👍